src/uncrustify.cpp

```diff
@@ -400,6 +403,9 @@ int main(int argc, char *argv[])
    // check keyword sort
    assert(keywords_are_sorted());
 
+   // Build options map
+   register_options();
```

Comment:
(Note: This needs to happen right away or there will be no options, which would cause even the basic help to report "There are currently **0** options and minimal documentation." This happens because we no longer statically know the option count... I don't *think* that's an issue, but if it is, we could change `make_options.py` compute it and stick it in `options.cpp` somewhere, though getting it as `constexpr` would be awkward.)